src/Compilers/CSharp/Portable/Utilities/ValueSetFactory.NintValueSet.cs

```diff
@@ -4,11 +4,11 @@
 
 #nullable enable
 
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
 namespace Microsoft.CodeAnalysis.CSharp
 {
-    using System.Text;
-    using Microsoft.CodeAnalysis.PooledObjects;
-    using Roslyn.Utilities;
     using static BinaryOperatorKind;
```

Comment:
Seen this pattern of putting ‘using static’ inside the namespace a few times. What advantages does this provide?

Reply:
Only the slight benefit that it doesn't have to be fully qualified.